src/client/js/CoreEditor.js

```diff
@@ -376,14 +414,27 @@ class LinkUrlControls extends React.Component {
     }
     render() {
         return (
-            <div style={styles.urlInputContainer}>
+            <div style={styles.linkTitleInputContainer}>
+                <label>Title: </label>
+                <input
+                  onChange={this.onTitleChange}
+                  ref="linkTitle"
+                  style={styles.linkTitleInput}
+                  type="text"
+                  value={this.props.titleValue}
+                  onKeyDown={this.onTitleInputKeyDown}
+                />
+                <label>Url: </label>
+                <input
+                  ref="url"
+                  style={styles.linkURLInput}
+                  type="text"
+                />
+                <label>Description: </label>
```

Comment:
Please use bootstrap for the forms, use something like this. 
```
             <div className="form-group">
                    <label htmlFor="title">Title:</label>
                    <input  id='title'
                            className="form-control"
                            onChange={this.onTitleChange}
                            ref="linkTitle"
                            type="text"
                            value={this.props.titleValue}
                            onKeyDown={this.onTitleInputKeyDown} />
                    </div>
                    <div className="form-group">
                        <label htmlFor="link">Link:</label>
                        <input  className='form-control'
                                id='link'
                                ref="url"
                                type="text" />
                 </div>
```

In this can you won't need using custom styles like linkURLInput etc